src/unittest/python/plugins/python/pep8_plugin_tests.py

```diff
@@ -17,14 +17,21 @@
 #   limitations under the License.
 
 from unittest import TestCase
+from pybuilder.core import Project
 from mock import Mock, patch
 from logging import Logger
 
-from pybuilder.plugins.python.pep8_plugin import check_pep8_available
+from pybuilder.plugins.python.pep8_plugin import (
+    check_pep8_available,
+    init_pep8_properties
+    )
 
 
 class CheckPep8AvailableTests(TestCase):
 
+    def setUp(self):
+        self.project = Project("basedir")
```

Comment:
This instance-level project is not used anywhere (`test_should_set_dependency` creates its own test double), so it should be removed.